### PR TITLE
Add github security policy

### PR DESCRIPTION
**Pull request by praminda**
### Purpose

Add a security policy to github. This will be available in https://github.com/wso2/product-microgateway/security/policy
### Issues

Only a task configuring github repository

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+| Version | Supported          |
+| ------- | ------------------ |
+| > 2.5.0 | :white_check_mark: |
+
+## Reporting Vulnerabilities
+
+WSO2 takes security issues very seriously. If you have any concerns regarding our product security or have uncovered a security vulnerability, we strongly encourage you to report that to our private and highly confidential security mailing list: security@wso2.com first, without disclosing them in any forums, sites or other groups - public or private. To protect the end-user security, these issues could be disclosed in other places only after WSO2 completes its Vulnerability Management Process.
+
+Warning : Please do not create GitHub issues for security vulnerabilities.
```
Comment (mathuriga) [inline .github/SECURITY.md]:
```suggestion
**Warning :** Please do not create GitHub issues for security vulnerabilities.
```

File: .github/SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| > 2.5.0 | :white_check_mark: |
+
+## Reporting Vulnerabilities
+
+WSO2 takes security issues very seriously. If you have any concerns regarding our product security or have uncovered a security vulnerability, we strongly encourage you to report that to our private and highly confidential security mailing list: security@wso2.com first, without disclosing them in any forums, sites or other groups - public or private. To protect the end-user security, these issues could be disclosed in other places only after WSO2 completes its Vulnerability Management Process.
```
Comment (mathuriga) [inline .github/SECURITY.md]: Shall we use HTML <a> Tag for "Vulnerability Management Process" with https://docs.wso2.com/display/Security/WSO2+Security+Vulnerability+Management+Process

File: .github/SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Warning : Please do not create GitHub issues for security vulnerabilities.
+
+WSO2 guidelines for reporting a security vulnerability page describes how to report a Security Vulnerability and includes a public key if you wish to send secure messages to security@wso2.com
```
Comment (mathuriga) [inline .github/SECURITY.md]: Shall we use HTML Tag for "WSO2 guidelines for reporting a security vulnerability" with https://docs.wso2.com/display/Security/WSO2+Security+Vulnerability+Reporting+Guidelines

File: .github/SECURITY.md (hunk 1 above)
Comment (praminda) [reply .github/SECURITY.md]: Fixed

File: .github/SECURITY.md (hunk 3 above)
Comment (praminda) [reply .github/SECURITY.md]: FIxed

File: .github/SECURITY.md (hunk 2 above)
Comment (praminda) [reply .github/SECURITY.md]: Fixed

**Pull request merged**